nanoFramework.Device.OneWire/OneWireMaster.cs

```diff
@@ -10,17 +10,52 @@
 namespace nanoFramework.Device.OneWire
 {
     /// <summary>
-    /// Represents a 1-Wire bus controller. The class provides methods and properties that an application can use to interact with the 1-Wire bus and connected devices.
+    /// Initializes a new instance of the <see cref="OneWireMaster"/> class.
+    /// Represents a 1-Wire bus master. The class provides methods and properties that an application can use to interact with the 1-Wire bus and connected devices.
     /// </summary>
-    public class OneWireController
+    public class OneWireMaster : IDisposable
```

Comment:
I would avoid the term 'Master' replace with Main or Principal or anything like this or even Controller.  Maybe even Host is the good word as it's the one used in the official documentation

Reply:
Well... that's the name that's used in 1-Wire.... I get the thing about the wording but....